.github/release-drafter.yml

```
@@ -1,5 +1,48 @@
 # Config for https://github.com/apps/release-drafter
-branches:
-  - master
+name-template: "$RESOLVED_VERSION 🦉 "
```

Previously, it'd generate draft releases with "Draft" title. Now, it can automatically resolve versions. To auto-increment, slap a "minor"/"major"/"patch" labels on PR and it'll auto-increment after merge.

---

Whoa...

---

```suggestion
name-template: "$RESOLVED_VERSION 🦉"
```

(minor whitespace :P)

---

Mostly a copy-paste issue with one that I was playing with on my private repo. :smile: We can leave it for now.